docs/org.eclipse.n4js.design/chapters/10_flowgraphs/flowgraphs.adoc

```diff
@@ -0,0 +1,434 @@
+////
+Copyright (c) 2017 NumberFour AG.
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v1.0
+which accompanies this distribution, and is available at
+http://www.eclipse.org/legal/epl-v10.html
+
+Contributors:
+  NumberFour AG - Initial API and implementation
+////
+
+[[chap:flowgraphs]]
+= Flow Graphs
+:find:
+
+
+
+[[sec:flowgraphs_overview]]
+[.language-n4js]
+== Flow graphs overview
+
+
+In this chapter, the control and data flow analyses are introduced.
+Since not all AST elements are relevant for the control or data flow analyses, a new marker class is introduced called `ControlFlowElement`.
+All AST elements which are part of the control flow graph implement this class.
+The term control flow is abbreviated as _CF_ and hence ``ControlFlowElement``s are abbreviated as _CF elements_.
+
+
+task::GH-120[]
```

Comment:
I have added the following line to config.adoc and the image to images in order to not get warnings during the generation:

```
:task_def_GH-: GitHub;GitHub Issues;https://github.com/eclipse/n4js/issues/{TASK_ID};issue.svg;GH-{TASK_ID}
```

Added that on your branch to make it easier.

Reply:
Thanks!